### ISSUE TITLE
Missing category: Hotels

### ISSUE THREAD

**Issue by sfkeller** [osmontrouge/caresteouvert#315]
**Is your feature request related to a problem? Please describe.**
In the category o the left, IMHO a category is missing which includes hotels, hostels, etc..

**Describe the solution you'd like**
Add category e.g. "Accommodation" which shows amenity=hotel, hostel, guest_house, motel.

**Describe alternatives you've considered**
* GOOG sticks with "Hotels".
* OsmAnd suggests "Tourism" which includes hotels: https://wiki.openstreetmap.org/wiki/OsmAnd#Examples_of_OSM_POI_categories_supported_in_OsmAnd

**Additional context**
* Finding POI categories isn't easy and there's no standard whatsoever about this. 
* See e.g. this data schema on how categories can be defined: https://github.com/geometalab/osmaxx/blob/master/docs/osmaxx_data_schema.md 

Soon following POI categories become of interest because of lockdown exit plans: hairdressser, garden centers, do-it-yourself-centers...

**Comment by vinber** [comment]
Hello,
do you have some examples of open hotel please ?

**Comment by PanierAvide** [comment]
Well, I think mostly all hotels stays open during lockdown, because some people live there for various reasons. I'm not sure this makes sense because either :
- You live in an hotel/hostel/... so you already know it is open
- You're looking for an hotel for travelling, so you don't know, however you shouldn't be travelling at start (except very specific cases)

Can you please precise in which case this could help general public ?

**Comment by sfkeller** [comment]
Do you really want me to enumerate hotels which are closed and some which are open? How many use cases of each category do you need in order to convince you :-O ?

**Comment by PanierAvide** [comment]
I'm not asking for list of open/closed hotels. I'm asking : who will we help if we show on our map hotels ? I see through context of France, and here we don't have apparent interest in showing them. However, elsewhere it can make sense. That's why I'm asking you for more context.

**Comment by sfkeller** [comment]
AFAIK this is still the situation in Switzerland: "Food stores, takeaway businesses, company canteens, food home delivery services and pharmacies will remain open, as will petrol stations, railway stations, banks, post offices, hotels, public administrations and social institutions. Maintenance facilities for means of transport may remain open." 

Source: https://www.bag.admin.ch/bag/en/home/das-bag/aktuell/medienmitteilungen.msg-id-78454.html (available also in FR and DE).

So basically all the above mentioned categories and subcategories are of great interest and are a great opportunity for this project and for OSM in general.

Many are covered here, but what seems still missing, are the category hotels and social institutions. Plus - as mentioned above - rather soon hairdressers, garden centers, do-it-yourself-centers.

**Comment by PanierAvide** [comment]
Thanks for this feedback. So we also have a technical issue regarding our current database structure, which doesn't keep `tourism=*` tags from OSM. We will work on this topic to broaden tags we keep so it makes easier to add other categories.

**Comment by sfkeller** [comment]
Merci a2.

**Comment by caugner** [comment]
@sfkeller Thanks for raising this. I wasn't aware of opened hotels either. Maybe we could also discuss this in the [bleibtoffen repository](https://github.com/bleibtoffen/bleibtoffen/)?

**Comment by sfkeller** [comment]
Yes, why not (bin zwar bis Mittwoch etwas absorbiert). We could perhaps also involve @simonpoole .

**Comment by francois2metz** [comment]
We now have the data in the database. Not sure what we should do. For which country/area Hotels should be included?

**Comment by caugner** [comment]
> We now have the data in the database. Not sure what we should do. For which country/area Hotels should be included?

AT, CH, DE: No (for now).

**Comment by francois2metz** [comment]
They are all open for business? Anyone can go to the hotels?

**Comment by simonpoole** [comment]
For CH: there are roughly 400 (of around a total of 5000) which are open, with some special cases that are only open for the military.

**Comment by caugner** [comment]
> For CH: there are roughly 400 (of around a total of 5000) which are open, with some special cases that are only open for the military.

Please open an issue in the repository for AT/CH/DE: https://github.com/bleibtoffen/bleibtoffen

**Comment by simonpoole** [comment]
> 
> 
> > For CH: there are roughly 400 (of around a total of 5000) which are open, with some special cases that are only open for the military.
> 
> Please open an issue in the repository for AT/CH/DE: https://github.com/bleibtoffen/bleibtoffen

An issue for what?

**Comment by caugner** [comment]
> > > For CH: there are roughly 400 (of around a total of 5000) which are open, with some special cases that are only open for the military.
> > 
> > 
> > Please open an issue in the repository for AT/CH/DE: https://github.com/bleibtoffen/bleibtoffen
> 
> An issue for what?

To request showing hotels in the map for any of these countries.

**Comment by caugner** [comment]
> AT, CH, DE: Don't show (for now).

Sorry, I mixed something up. Hotels should probably shown in `CH`, but we'll discuss this first.

**Comment by vinber** [comment]
a hotel with opening_hours:covid19=open can be see on the map now.
An example in france https://www.caresteouvert.fr/other@44.783553,-0.587467,17.08/place/w139675721.
however actually hotel without opening_hours:covid19=* are not display in france.

thanks to key tourism=* @francois2metz @PanierAvide @vdct

**Comment by PanierAvide** [comment]
I added an icon + label for hotel, so the ones appearing will be shown properly.

**Comment by PanierAvide** [comment]
Closing the issue, if we want to go further (show all hotels in a country for example), don't hesitate to reopen the issue.

**Comment by simonpoole** [comment]
The problem I see is as that there is no corresponding category there is no way for the owner/operator of a hotel to actually find their establishment and, if necessary, add the tags.

**Comment by sfkeller** [comment]
Hotels really should get displayed - at least in Switzerland - because they _can_ stay open there. 

So in order to insert or update the status of a Hotel there's no other way than to first find and select it in this app. 

Pls. reopen this issue.
